moveit_core/robot_model/src/planar_joint_model.cpp

```diff
@@ -139,39 +141,109 @@ void PlanarJointModel::getVariableRandomPositionsNearBy(random_numbers::RandomNu
   normalizeRotation(values);
 }
 
+void computeTurnDriveTurnGeometry(const double* from, const double* to, double& dx, double& dy, double& initial_turn,
+                                  double& drive_angle, double& final_turn)
+{
+  dx = to[0] - from[0];
+  dy = to[1] - from[1];
+  drive_angle = atan2(dy, dx);
+
+  initial_turn = angles::shortest_angular_distance(from[2], drive_angle);
+  drive_angle = from[2] + initial_turn;
+  final_turn = to[2] - drive_angle;
```

Comment:
don't you need angles::shortest_angular_distance here as well?

Reply:
No, and I should add comments to the code explicitly stating why.